src/_guides/language/language-tour.md

```diff
@@ -489,22 +487,23 @@ units.
 You can concatenate strings using adjacent string literals or the `+`
 operator:
 
-<!-- adjacent_string_literals.dart -->
+<?code-excerpt "test/language_tour/built_in_types_test.dart (adjacent-string-literals)"?>
 {% prettify dart %}
-var s1 = 'String ' 'concatenation'
-         " works even over line breaks.";
-assert(s1 == 'String concatenation works even over '
-             'line breaks.');
+var s1 = 'String '
+    'concatenation'
+    " works even over line breaks.";
+assert(s1 ==
+    'String concatenation works even over '
+    'line breaks.');
```

Comment:
I can't seem to coerce *dartfmt* to give us the same formatting as before. If you think that it is important to preserve the original formatting, let me know and I'll add a few lines of custom perl to fix it after *dartfmt* has been applied to all the sources.

Btw, the formatting script is `scripts/dartfmt.sh`.

Reply:
This is fine.